frontend/pages/review.tsx

```diff
@@ -35,29 +37,88 @@ function ErrorWrapper({ message }: { message: string | undefined }) {
 }
 
 function ApprovalList({ category, filter }: { category: ApprovalCategory; filter: ApprovalFilterType }) {
+  const { approvals, isApprovalsLoading, isApprovalsError, mutateApprovals } = useListApprovals(category, filter)
+
+  const error = MultipleErrorWrapper(
+    `Unable to load approval page`,
+    {
+      isApprovalsError,
+    },
+    ErrorWrapper
+  )
+  if (error) return error
+
+  const getUploadCategory = (approvalCategory: ApprovalCategory) =>
```

Comment:
I feel like `UploadCategory` should be a specified type as it's the sort of thing we're likely to want to use in future. It's nice to help future devs to be consistent with the terminology being defined here.

I suggest making a new type in `bailo/frontend/types/types.ts`:
```
export type UploadCategory = 'model' | 'deployment'
```

You can then update `getUploadCategory`. I've changed it from being a regular function to `useMemo`, so it should be moved above any conditional logic in this component. I've also updated it to use `ApprovalCategory` from the correct file (`bailo/frontend/types/types.ts`), so your import statement at the top will need updating:
```
const uploadCategory: UploadCategory = useMemo(() => approvalCategory === ApprovalCategory.Upload ? 'model' : 'deployment', [approvalCategory])
```